### PR TITLE
updated cms and hosting requirements

### PR DESCRIPTION
**Pull request by alina-tincas**
## Description

_What did you add/update/change?_
Requirements page with links to Microsoft-supported versions (as Windows 2012 is currently EOL and we follow the Microsoft version release cadence)

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
CMS v10,11,12

## Deadline (if relevant)
asap

_When should the content be published?_
anytime

### CODE REVIEW

File: 10/umbraco-cms/fundamentals/setup/requirements.md [hunk 1]
```diff
@@ -11,95 +11,49 @@ The Umbraco UI should work in all modern browsers:
...
-* IIS 10 and higher
-* SQL Server 2019 and higher
-* .NET 6.0.5+
-* Ability to set file permissions to include create/read/write (or better) for the user that "owns" the Application Pool for your site (NETWORK SERVICE, typically)
-{% endtab %}
+* [Windows Supported releases](https://learn.microsoft.com/en-us/dotnet/core/install/windows?tabs=net70#supported-releases)
+* [MacOs Supported releases](https://learn.microsoft.com/en-us/dotnet/core/install/macos#supported-releases), [Ubuntu Supported distributions](https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#supported-distributions) and other [Linux Packages](https://learn.microsoft.com/en-us/dotnet/core/install/linux#packages)
+* [.NET Supported releases](https://dotnet.microsoft.com/en-us/platform/support/policy)
+* [IIS Supported releases](https://learn.microsoft.com/en-us/lifecycle/products/internet-information-services-iis)
+* [SQL Server Supported releases](https://learn.microsoft.com/en-us/sql/sql-server/end-of-support/sql-server-end-of-support-overview?view=sql-server-ver16#lifecycle-dates) and [SQLite](https://www.sqlite.org/index.html)
```
Comment (jonat123) [inline 10/umbraco-cms/fundamentals/setup/requirements.md:42]:
```suggestion
* [Windows Supported releases](https://learn.microsoft.com/en-us/dotnet/core/install/windows?tabs=net70#supported-releases)
* [MacOs Supported releases](https://learn.microsoft.com/en-us/dotnet/core/install/macos#supported-releases)
* [Ubuntu Supported distributions](https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#supported-distributions) and other [Linux Packages](https://learn.microsoft.com/en-us/dotnet/core/install/linux#packages)
* [.NET Supported releases](https://dotnet.microsoft.com/en-us/platform/support/policy)
* [IIS Supported releases](https://learn.microsoft.com/en-us/lifecycle/products/internet-information-services-iis)
* [SQL Server Supported releases](https://learn.microsoft.com/en-us/sql/sql-server/end-of-support/sql-server-end-of-support-overview?view=sql-server-ver16#lifecycle-dates)
* [SQLite](https://www.sqlite.org/index.html)
```

File: 11/umbraco-cms/fundamentals/setup/requirements.md [hunk 2]
```diff
@@ -13,47 +13,43 @@ The Umbraco UI works in all modern browsers:
...
+* [Windows Supported releases](https://learn.microsoft.com/en-us/dotnet/core/install/windows?tabs=net70#supported-releases)
+* [MacOs Supported releases](https://learn.microsoft.com/en-us/dotnet/core/install/macos#supported-releases), [Ubuntu Supported distributions](https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#supported-distributions) and other [Linux Packages](https://learn.microsoft.com/en-us/dotnet/core/install/linux#packages)
+* [.NET Supported releases](https://dotnet.microsoft.com/en-us/platform/support/policy)
+* [IIS Supported releases](https://learn.microsoft.com/en-us/lifecycle/products/internet-information-services-iis)
+* [SQL Server Supported releases](https://learn.microsoft.com/en-us/sql/sql-server/end-of-support/sql-server-end-of-support-overview?view=sql-server-ver16#lifecycle-dates) and [SQLite](https://www.sqlite.org/index.html)
+
+_For more information, see the_ [_Host and deploy ASP.NET Core applications_](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/?view=aspnetcore-7.0) _article in the Microsoft documentation._
```
Comment (jonat123) [inline 11/umbraco-cms/fundamentals/setup/requirements.md:44]:
```suggestion
* [Windows Supported releases](https://learn.microsoft.com/en-us/dotnet/core/install/windows?tabs=net70#supported-releases)
* [MacOs Supported releases](https://learn.microsoft.com/en-us/dotnet/core/install/macos#supported-releases)
* [Ubuntu Supported distributions](https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#supported-distributions) and other [Linux Packages](https://learn.microsoft.com/en-us/dotnet/core/install/linux#packages)
* [.NET Supported releases](https://dotnet.microsoft.com/en-us/platform/support/policy)
* [IIS Supported releases](https://learn.microsoft.com/en-us/lifecycle/products/internet-information-services-iis)
* [SQL Server Supported releases](https://learn.microsoft.com/en-us/sql/sql-server/end-of-support/sql-server-end-of-support-overview?view=sql-server-ver16#lifecycle-dates)
* [SQLite](https://www.sqlite.org/index.html)

_For more information, see the_ [_Host and deploy ASP.NET Core applications_](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/?view=aspnetcore-7.0) _article in the Microsoft documentation._
```

File: 12/umbraco-cms/fundamentals/setup/requirements.md [hunk 3]
```diff
@@ -13,47 +13,43 @@ The Umbraco UI works in all modern browsers:
...
+* [Windows Supported releases](https://learn.microsoft.com/en-us/dotnet/core/install/windows?tabs=net70#supported-releases)
+* [MacOs Supported releases](https://learn.microsoft.com/en-us/dotnet/core/install/macos#supported-releases), [Ubuntu Supported distributions](https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#supported-distributions) and other [Linux Packages](https://learn.microsoft.com/en-us/dotnet/core/install/linux#packages)
+* [.NET Supported releases](https://dotnet.microsoft.com/en-us/platform/support/policy)
+* [IIS Supported releases](https://learn.microsoft.com/en-us/lifecycle/products/internet-information-services-iis)
+* [SQL Server Supported releases](https://learn.microsoft.com/en-us/sql/sql-server/end-of-support/sql-server-end-of-support-overview?view=sql-server-ver16#lifecycle-dates) and [SQLite](https://www.sqlite.org/index.html)
+
+_For more information, see the_ [_Host and deploy ASP.NET Core applications_](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/?view=aspnetcore-7.0) _article in the Microsoft documentation._
```
Comment (jonat123) [inline 12/umbraco-cms/fundamentals/setup/requirements.md:44]:
```suggestion
* [Windows Supported releases](https://learn.microsoft.com/en-us/dotnet/core/install/windows?tabs=net70#supported-releases)
* [MacOs Supported releases](https://learn.microsoft.com/en-us/dotnet/core/install/macos#supported-releases)
* [Ubuntu Supported distributions](https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu#supported-distributions) and other [Linux Packages](https://learn.microsoft.com/en-us/dotnet/core/install/linux#packages)
* [.NET Supported releases](https://dotnet.microsoft.com/en-us/platform/support/policy)
* [IIS Supported releases](https://learn.microsoft.com/en-us/lifecycle/products/internet-information-services-iis)
* [SQL Server Supported releases](https://learn.microsoft.com/en-us/sql/sql-server/end-of-support/sql-server-end-of-support-overview?view=sql-server-ver16#lifecycle-dates)
* [SQLite](https://www.sqlite.org/index.html)

_For more information, see the_ [_Host and deploy ASP.NET Core applications_](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/?view=aspnetcore-7.0) _article in the Microsoft documentation._
```

**Comment by alina-tincas** [comment]
Thank you for the review and suggestions 🙌 I have committed those suggestions! 

Then am merging this in 💪

**Pull request merged**